internal/jobcontainers/jobcontainer.go

```diff
@@ -126,18 +140,39 @@ func Create(ctx context.Context, id string, s *specs.Spec) (_ cow.Container, _ *
 		}
 	}()
 
-	sandboxPath := fmt.Sprintf(sandboxMountFormat, id)
-	if err := mountLayers(ctx, id, s, sandboxPath); err != nil {
-		return nil, nil, errors.Wrap(err, "failed to mount container layers")
-	}
-	container.sandboxMount = sandboxPath
+	// Check if we support file binding once to avoid needing to stat for the dll on
+	// every container creation.
+	checkBindSupportOnce.Do(func() {
+		bindDLL := `C:\windows\system32\bindfltapi.dll`
```

Comment:
Is there any case where we would want to support a different path for this?

Reply:
I guess something to note is this dll has a different name on 19h1 (just bindflt.dll), but I don't think we had any plans to support this work on that version. In terms of some case where we expect bindfltapi to be present elsewhere, no not that I can think of at least

Reply:
I think that's fine to not support 19H1, pretty sure it's out of support now.